src/rdf4cpp/storage/NodeStorage.cpp

```diff
@@ -1,16 +1,14 @@
-#include "NodeStorage.hpp"
-
+#include <rdf4cpp/storage/NodeStorage.hpp>
 #include <rdf4cpp/storage/reference_node_storage/SyncReferenceNodeStorage.hpp>
 
 namespace rdf4cpp::storage {
 
+namespace reference_node_storage {
+SyncReferenceNodeStorage default_instance{};
+} // reference_node_storage
+
 #ifndef DOXYGEN_PARSER
-reference_node_storage::SyncReferenceNodeStorage default_node_storage_holder_;
-DynNodeStoragePtr default_node_storage{default_node_storage_holder_};
+DynNodeStoragePtr default_node_storage{reference_node_storage::default_instance};
 #endif // DOXYGEN_PARSER
 
-void reset_default_node_storage() noexcept {
-    default_node_storage = default_node_storage_holder_;
-}
```

Comment:
Is this function not needed anymore?

Reply:
To achieve the same effect, you can now do
```c++
rdf4cpp::storage::default_node_storage = rdf4cpp::storage::reference_node_storage::default_instance;
```